packages/story-editor/src/components/panels/panel/shared/title.js

```diff
@@ -111,17 +110,16 @@ const Collapse = styled.button`
   cursor: pointer;
   margin-left: -12px;
   transition: ${BUTTON_TRANSITION_TIMING};
-
   &:hover,
-  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR}, &[${ThemeGlobals.FOCUS_VISIBLE_DATA_ATTRIBUTE}] {
+   &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR}, &[${ThemeGlobals.FOCUS_VISIBLE_DATA_ATTRIBUTE}] {
```

Comment:
How did this indentation happen here?

Reply:
our pre-commit hook style lint step has been misformatting some files. typically it's on random files you don't touch and it just completely breaks them. To fix I normally go back and undo the change and commit the reverts with a `--no-verify` so that the style lint step doesn't break anything again.

This indent happened as a result of that.

The misformatting is still intermittent, but it typically happens on files that are not part of your PR so I don't worry too much about running the pre-commit hook on those files